demos/tableOfContents/index.html

```diff
@@ -39,68 +39,85 @@
             <nav class="row">
                 <div class="flc-toc-tocContainer demo-toc large-4 small-centered columns"></div>
             </nav>
-
-            <h1>Amphibians</h1>
+            <h1>Species</h1>
+            <h2>Amphibians</h2>
```

Comment:
As part of the a11y review we had for UIO it was mentioned that it is atypical to have multiple `<h1>`'s on a page. To avoid any issues in this regard for the demo, i just made a new top level `<h1>` and dropped everything down a level. This did not affect the tests because it is the demo, not the tested html file.